UploadServlet.java

```diff
@@ -72,86 +114,96 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 				dir.mkdir();
 			}
 			
-			String author = request.getParameter("author");
-			String subject = request.getParameter("subject");
-			String title = request.getParameter("title");
-			String highlights = request.getParameter("highlights");
-			String abstracts = request.getParameter("abstracts");
 			Timestamp time = new Timestamp(System.currentTimeMillis());
 			
 			
 			Part filePart = request.getPart("file");
 			String fileName = filePart.getSubmittedFileName();
 			String path = folderName + File.separator + fileName;
+			String fileType = filePart.getContentType();// get the uploaded file type
 			
-//			System.out.println("fileNmae:" +fileName);
-//			System.out.println("Path: "+ uploadPath);
-			
-			InputStream is = filePart.getInputStream();
-			Files.copy(is, Paths.get(uploadPath + File.separator + fileName), StandardCopyOption.REPLACE_EXISTING);
-			
-			try
-			{
-				Class.forName("com.mysql.jdbc.Driver");
+			// if the uploaded file type is PDF, then upload the file to the server
+			if(fileType == "application/pdf") {
 				
-				String u = "root";
-				String password = "Fathihachellam.1";
-				String ul = "jdbc:mysql://localhost:3306/ooad?autoReconnect=true&&useSSL=false";
+				InputStream is = filePart.getInputStream();
+				Files.copy(is, Paths.get(uploadPath + File.separator + fileName), StandardCopyOption.REPLACE_EXISTING);
 				
-				con = DriverManager.getConnection(ul, u, password);
-				
-				String sql = "insert into article(subject, title, highlight, abstracts, author, time, filename, path) values (?,?,?,?,?,?,?,?)";
-				
-				ps = con.prepareStatement(sql);
-				
-				ps.setString(1, subject);
-				ps.setString(2, title);
-				ps.setString(3, highlights);
-				ps.setString(4, abstracts);
-				ps.setString(5, author);
-				ps.setTimestamp(6, time);
-				ps.setString(7, fileName);
-				ps.setString(8, path);
-				
-				int status = ps.executeUpdate();
+				try
+				{
+					Class.forName("com.mysql.jdbc.Driver");
+					
+					String u = "root";
+					String password = "124536";

```

Comment:
@hema-001 

Is it a good idea to include password 124536 in the source code?

**Hui**

Reply:
No it's not.

But, we are pretty naive when it comes to security considerations.

A good insight from you might be helpful.